[2022-02-09 09:01:33,915] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.bigquery_external_table_task scheduled__2021-11-01T23:00:00+00:00 [queued]>
[2022-02-09 09:01:33,932] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.bigquery_external_table_task scheduled__2021-11-01T23:00:00+00:00 [queued]>
[2022-02-09 09:01:33,932] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 09:01:33,933] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-09 09:01:33,933] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 09:01:33,954] {taskinstance.py:1259} INFO - Executing <Task(BigQueryCreateExternalTableOperator): bigquery_external_table_task> on 2021-11-01 23:00:00+00:00
[2022-02-09 09:01:33,963] {standard_task_runner.py:52} INFO - Started process 612 to run task
[2022-02-09 09:01:33,976] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yelloTaXi_HW_data_ingestee_gcs_dag', 'bigquery_external_table_task', 'scheduled__2021-11-01T23:00:00+00:00', '--job-id', '526', '--raw', '--subdir', 'DAGS_FOLDER/yelloTaXi_data_ingestee_gcs_dag.py', '--cfg-path', '/tmp/tmpq65ur0fa', '--error-file', '/tmp/tmpk2de9itv']
[2022-02-09 09:01:33,978] {standard_task_runner.py:77} INFO - Job 526: Subtask bigquery_external_table_task
[2022-02-09 09:01:34,081] {logging_mixin.py:109} INFO - Running <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.bigquery_external_table_task scheduled__2021-11-01T23:00:00+00:00 [running]> on host cc021fb126ed
[2022-02-09 09:01:34,153] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-09 09:01:34,191] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yelloTaXi_HW_data_ingestee_gcs_dag
AIRFLOW_CTX_TASK_ID=bigquery_external_table_task
AIRFLOW_CTX_EXECUTION_DATE=2021-11-01T23:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-11-01T23:00:00+00:00
[2022-02-09 09:01:34,193] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2022-02-09 09:01:34,835] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=yelloTaXi_HW_data_ingestee_gcs_dag, task_id=bigquery_external_table_task, execution_date=20211101T230000, start_date=20220209T090133, end_date=20220209T090134
[2022-02-09 09:01:34,898] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-09 09:01:34,954] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-02-09 09:17:24,417] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.bigquery_external_table_task scheduled__2021-11-01T23:00:00+00:00 [queued]>
[2022-02-09 09:17:24,435] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.bigquery_external_table_task scheduled__2021-11-01T23:00:00+00:00 [queued]>
[2022-02-09 09:17:24,435] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 09:17:24,435] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-09 09:17:24,435] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 09:17:24,506] {taskinstance.py:1259} INFO - Executing <Task(BigQueryCreateExternalTableOperator): bigquery_external_table_task> on 2021-11-01 23:00:00+00:00
[2022-02-09 09:17:24,516] {standard_task_runner.py:52} INFO - Started process 776 to run task
[2022-02-09 09:17:24,532] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yelloTaXi_HW_data_ingestee_gcs_dag', 'bigquery_external_table_task', 'scheduled__2021-11-01T23:00:00+00:00', '--job-id', '632', '--raw', '--subdir', 'DAGS_FOLDER/yelloTaXi_data_ingesteee_gcs_dag.py', '--cfg-path', '/tmp/tmpkdhldwiv', '--error-file', '/tmp/tmp76r7tth1']
[2022-02-09 09:17:24,533] {standard_task_runner.py:77} INFO - Job 632: Subtask bigquery_external_table_task
[2022-02-09 09:17:24,874] {logging_mixin.py:109} INFO - Running <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.bigquery_external_table_task scheduled__2021-11-01T23:00:00+00:00 [running]> on host 75953ff4f3d5
[2022-02-09 09:17:25,292] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-09 09:17:25,321] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yelloTaXi_HW_data_ingestee_gcs_dag
AIRFLOW_CTX_TASK_ID=bigquery_external_table_task
AIRFLOW_CTX_EXECUTION_DATE=2021-11-01T23:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-11-01T23:00:00+00:00
[2022-02-09 09:17:25,323] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2022-02-09 09:17:26,051] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=yelloTaXi_HW_data_ingestee_gcs_dag, task_id=bigquery_external_table_task, execution_date=20211101T230000, start_date=20220209T091724, end_date=20220209T091726
[2022-02-09 09:17:26,507] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-09 09:17:26,659] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
